scripts/globals/combat/magic_hit_rate.lua

```diff
@@ -271,26 +281,32 @@ xi.combat.magicHitRate.calculateResistRate = function(actor, target, skillType,
         resistRank = target:getMod(elementTable[spellElement][4])
     end
 
+    -- Handles increasing res rank if a skillchain is present
+    local _, skillchainCount = FormMagicBurst(spellElement, target)
+
+    if skillchainCount > 0 then
+        resistRank = utils.clamp(resistRank - 1, -3, 11)
+    end
```

Comment:
I dont think this is needed. Caster already gets a magic accuracy bonus in the macc step